10.Mender-server/docs.md

```diff
@@ -4135,6 +4141,16 @@ New changes in integration since 3.3.0:
 
 _Released 03.10.2023_
 
+### Security fixes
```

Comment:
Having a separate section here looks very good. Nice initiative.

Could you please extend it also to 3.7.4 ([example](https://github.com/mendersoftware/mender-docs-changelog/blob/2079d62fdd53fc1687d717b97e67f9e4c40f2756/10.Mender-server/docs.md#bug-fixes-3)) and 7.6.4 changelogs?

Reply:
Done, updated 3.6.4 and 3.7.4 in the latest commit.